<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUaModbusDataBlockWidget</class>
 <widget class="QWidget" name="QUaModbusDataBlockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <spacer name="horizontalSpacer_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>274</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushButtonApply">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QUaModbusDataBlockWidgetStatus" name="widgetBlockStatus" native="true"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QUaModbusDataBlockWidgetEdit" name="widgetBlockEdit" native="true"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonAddValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add Value</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="Line" name="line3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonPerms">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Permissions</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonClear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Clear Values</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>193</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUaModbusDataBlockWidgetEdit</class>
   <extends>QWidget</extends>
   <header location="global">QUaModbusDataBlockWidgetEdit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QUaModbusDataBlockWidgetStatus</class>
   <extends>QWidget</extends>
   <header location="global">QUaModbusDataBlockWidgetStatus</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
